Luminescent/src/astechzgo/luminescent/entity/Entity.java

```diff
@@ -1,5 +1,7 @@
 package astechzgo.luminescent.entity;
 
 public interface Entity {
-
+public void SetHealth(int health);
```

Comment:
These are things that only affect mobs, not all entities.  You could make a mob class that extends entity to do this if you want, but I don't think it should be in entity.